src/lib/updating-element.ts

```diff
@@ -736,10 +736,34 @@ export abstract class UpdatingElement extends HTMLElement {
    * rendered element before fulfilling this Promise. To do this, first await
    * `super.updateComplete` then any subsequent state.
    *
+   * IMPORTANT: Do not override this getter directly. Override the
```

Comment:
This is only import for TypeScript users, yes? Babel does the right thing it seems. Unsure about Closure.

Reply:
Yes, but I was thinking it's preferable to have just one style recommendation that will always be safe.

Reply:
I guess. I just find it a little odd that this is really only because of a very specific set of circumstances triggering a bug in one compiler with one particular setting, and only when subclassers themselves are using TypeScript...

Oh, to be done with ES5.